kartothek/io/dask/dataframe.py

```diff
@@ -301,3 +307,94 @@ def update_dataset_from_ddf(
         metadata=metadata,
         metadata_merger=metadata_merger,
     )
+
+
+def collect_dataset_metadata(
+    store,
+    dataset_uuid,
+    table_name=SINGLE_TABLE,
+    predicates=None,
+    frac=1.0,
+    factory=None,
+):
+    """
+    Collect parquet metadata of the dataset. The `frac` parameter can be used to select a subset of the data.
+
+    .. warning::
+      If the size of the partitions is not evenly distributed, e.g. some partitions might be larger than others,
+      the metadata returned is not a good approximation for the whole dataset metadata.
+    .. warning::
+      Using the `frac` parameter is not encouraged for a small number of total partitions.
+
+
+    Parameters
+    ----------
+    store
+      A factory function providing a KeyValueStore
+    dataset_uuid
+      The dataset's unique identifier
+    table_name
+      Name of the kartothek table for which to retrieve the statistics
+    predicates
+      Kartothek predicates to apply filters on the data for which to gather statistics
+
+      .. warning::
+          Filtering will only be applied for predicates on indices.
+          The evaluation of the predicates therefore will therefore only return an approximate result.
+
+    frac
+      Fraction of the total number of partitions to use for gathering statistics. `frac == 1.0` will use all partitions.
+    factory
+       A DatasetFactory holding the store and UUID to the source dataset.
+
+    Returns
+    -------
+    A delayed pandas.DataFrame containing the following information about dataset statistics:
```

Comment:
This is not consistent with output (currently it's either a dask.DataFrame or dask.delayed object containing a pandas.DataFrame)